tools/release/main.go

```diff
@@ -96,16 +96,16 @@ func generateFile(name, version string) error {
 		PreviousReleasesLink: "https://github.com/mongodb/mongodb-atlas-cli/atlascli/releases",
 		ReleaseNotesLink:     "https://dochub.mongodb.org/core/atlas-cli-changelog",
 		TutorialLink:         "https://dochub.mongodb.org/core/install-atlas-cli",
-		Platform: []Platform{
-			*newPlatform(version, "x86_64", "linux", "Linux (x86_64)", []string{"tar.gz"}),
-			*newPlatform(version, "arm64", "linux", "Linux (arm64)", []string{"tar.gz"}),
-			*newPlatform(version, "x86_64", "linux", "Debian 10, 11, 12 / Ubuntu 20.04, 22.04, 24.04 (x86_64)", []string{"deb"}),
-			*newPlatform(version, "arm64", "linux", "Debian 10, 11, 12 / Ubuntu 20.04, 22.04, 24.04 (arm64)", []string{"deb"}),
-			*newPlatform(version, "x86_64", "linux", "Red Hat + CentOS 7, 8, 9 / SUSE 12 + 15 / Amazon Linux 2, 2023 (x86_64)", []string{"rpm"}),
-			*newPlatform(version, "arm64", "linux", "Red Hat + CentOS 7, 8, 9 / SUSE 12 + 15 / Amazon Linux 2, 2023 (arm64)", []string{"rpm"}),
-			*newPlatform(version, "x86_64", "windows", "Microsoft Windows", []string{"msi", "zip"}),
-			*newPlatform(version, "x86_64", "macos", "macOS (x86_64)", []string{"zip"}),
-			*newPlatform(version, "arm64", "macos", "macOS (arm64)", []string{"zip"}),
+		Platform: []*Platform{
+			newPlatform(version, "x86_64", "linux", "Linux (x86_64)", []string{"tar.gz"}),
+			newPlatform(version, "arm64", "linux", "Linux (arm64)", []string{"tar.gz"}),
+			newPlatform(version, "x86_64", "linux", "Debian 11, 12 / Ubuntu 20.04, 22.04, 24.04 (x86_64)", []string{"deb"}),
+			newPlatform(version, "arm64", "linux", "Debian 11, 12 / Ubuntu 20.04, 22.04, 24.04 (arm64)", []string{"deb"}),
+			newPlatform(version, "x86_64", "linux", "Red Hat + CentOS 8, 9 / SUSE 12 + 15 / Amazon Linux 2023 (x86_64)", []string{"rpm"}),
+			newPlatform(version, "arm64", "linux", "Red Hat + CentOS 8, 9 / SUSE 12 + 15 / Amazon Linux 2023 (arm64)", []string{"rpm"}),
```

Comment:
Could you create a Jira ticket and add the release note section to clarify that the next release will not include these OSe?

Reply:
this OSes were removed in 1.25 see https://www.mongodb.com/docs/atlas/cli/current/atlas-cli-changelog/#atlas-cli-1.25.0 but when they were dropped we never updated this file